.github/workflows/build.yml

```diff
@@ -89,6 +93,8 @@ jobs:
       - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84 # v2.7.3
         with:
           key: ${{ matrix.target_os }}-${{ matrix.arch }}
+      - name: install python3-requests
+        run: apt update && apt install -y python3-requests
```

Comment:
I tought this was added to dockers itself, do we need it here?

Also why `apt update` here, but not in other places?

Reply:
Sames as https://github.com/NordSecurity/libtelio/pull/543#discussion_r1608039724

It is isolated environment, and it prohibits u to use `pip` for some reason ...

Reply:
Fixed